ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -46,6 +51,10 @@ common common-lib
     ghc-options: -fno-ignore-asserts
     cpp-options: -DENABLE_ASSERTIONS
 
+  if flag(expensive-invariants)
+    cpp-options:
+      -DCHECK_NOTHUNKS_INVARIANT
```

Comment:
> Will there ever be more than one expensive-invariant? If not, I'd rather consolidate the build and CPP flag under the "same" name (either checknothunksinvariants/CHECK_NOTHUNK_INVARIANTS, or expensive-invariants/EXPENSIVE_INVARIANTS).

Good point, I think I would be weakly in favor of the latter option. I hope we won't need to introduce additional levels of differentiation here :sweat_smile: